plugins/experimental/tls_bridge/CMakeLists.txt

```diff
@@ -17,5 +17,5 @@
 
 add_atsplugin(tls_bridge tls_bridge.cc)
 
-target_link_libraries(tls_bridge PRIVATE ts::tsutil libswoc::libswoc)
+target_link_libraries(tls_bridge PRIVATE libswoc::libswoc PCRE::PCRE)
```

Comment:
Please mark `PCRE::PCRE` linkage as transitive so it can be found for cleanup later.